Gemfile

```diff
@@ -2,14 +2,17 @@ source "https://rubygems.org"
 
 gemspec
 
+# for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
```

Comment:
TBH, I haven't investigated translation yet. It's on the list to figure out based on other hammer repos. Would you rather it be in this PR or a separate?